hooks/post-up

```diff
@@ -6,4 +6,15 @@ if [ ! -e $HOME/.vim/autoload/plug.vim ]; then
   curl -fLo $HOME/.vim/autoload/plug.vim --create-dirs \
       https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim
 fi
+
 vim -u $HOME/.vimrc.bundles +PlugInstall +PlugClean! +qa
+
+if grep --quiet --no-messages path_helper /etc/zprofile; then
```

Comment:
The more portable way to write this is:

```sh
if grep path_helper /etc/zprofile >/dev/null 2>/dev/null; then
```

That will work on unix systems in addition to GNU.